server-spi-private/src/main/java/org/keycloak/models/oid4vci/CredentialScopeModel.java

```diff
@@ -114,11 +114,30 @@ public class CredentialScopeModel implements ClientScopeModel {
     public static final String TOKEN_JWS_TYPE = "vc.credential_build_config.token_jws_type";
 
     /**
-     * this configuration property can be used to enforce specific claims to be included in the metadata, if they
-     * would normally not and vice versa
+     * this configuration property can be used to enforce specific claims to be included in the metadata, if they would
+     * normally not and vice versa
      */
     public static final String INCLUDE_IN_METADATA = "vc.include_in_metadata";
 
+    /**
+     * OPTIONAL. Object that describes the requirement for key attestations as described in Appendix D, which the
+     * Credential Issuer expects the Wallet to send within the proof(s) of the Credential Request. If the Credential
+     * Issuer does not require a key attestation, this parameter MUST NOT be present in the metadata. If both
+     * key_storage and user_authentication parameters are absent, the key_attestations_required parameter may be empty,
+     * indicating a key attestation is needed without additional constraints.
+     */
+    public static final String KEY_ATTESTATION_REQUIRED = "vc.key_attestation_required";
```

Comment:
Can we name this `vc.key_attestations_required` for consistency with the naming in the spec?
```suggestion
    public static final String KEY_ATTESTATION_REQUIRED = "vc.key_attestations_required";
```